packages/terra-functional-testing/lib/services/wdio-asset-service.js

```diff
@@ -0,0 +1,45 @@
+const ExpressServer = require('../express-server/express-server');
+const WebpackServer = require('../webpack-server/webpack-server');
+const Logger = require('../logger/logger');
+
+const logger = new Logger({ prefix: 'wdio-asset-service' });
+
+class AssetService {
```

Comment:
To me, AssetService doesn't seem representative of the services' intent.... ServerService maybe? TerraServerService?     I can't say those are great. Since these toggle two flavors of static servers. It's almost like we should provide a default webpack config when a site is provided to use the [Copy Webpack Plugin](https://webpack.js.org/plugins/copy-webpack-plugin/) and serve these pre-compiles files with webpack-dev-server as well. 🤷‍♀️

Reply:
I renamed this a few times before pushing out the PR, picking a descriptive name has been challenging. I have a comment in the summary asking for suggestions, hopefully we find something fitting

Reply:
TerraServerService, TestServerService, AssetServerService.

What were other names you previously had?

Reply:
maybe we do a poll?

Reply:
Maybe TerraServerService or TerraAssetServerService?

Reply:
While/as long as this service is inside of `terra-functional-testing` I don't think the `terra` prefix adds a lot of value. A few more names to drop into the options list:

- StaticAssetService
- ExpressService // Assuming it's possible to consolidate the two services
- WebpackService // Assuming it's possible to consolidate the two services

`Server` feels a little misleading as well, although there is indeed a server being launched, it doesn't make it obvious what its purpose is, the `AssetServerService` suggestion is more clear about the functionality of the service

Reply:
Renamed to `AssetServerService`:

https://github.com/cerner/terra-toolkit/pull/417/commits/5427b46718561ff890cc508fcad2a3be61bb2998